plugins/UsersManager/API.php

```diff
@@ -709,11 +876,11 @@ public function updateUser($userLogin, $password = false, $email = false, $alias
     }
 
     /**
-     * Delete a user and all its access, given its login.
+     * Delete one or more users and all its access, given its login.
      *
-     * @param string $userLogin the user login.
+     * @param string|string[] $userLogin the user login(s).
```

Comment:
Could we keep the API simple by using a bulk API request and request `deleteUser` for each userLogin?